File: core/src/main/java/com/ataulm/whatsnext/ApiMemberAccountResponse.java
package com.ataulm.whatsnext;

import com.google.gson.annotations.SerializedName;

class ApiMemberAccountResponse {

    @SerializedName("member")
    Member member;

    static class Member {
        @SerializedName("id")
        String letterboxId;

        @SerializedName("displayName")
        String displayName;
    }
}

File: letterboxd/src/main/java/com/ataulm/letterboxd/ApiMemberAccountResponse.kt
package com.ataulm.letterboxd

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class ApiMemberAccountResponse(

    @field:Json(name = "member")
    val member: Member
) {

    @JsonClass(generateAdapter = true)
    class Member(
        @field:Json(name = "id")
        val letterboxId: String,

        @field:Json(name = "displayName")
        val displayName: String
    )
}
